README.md

```diff
@@ -27,17 +27,33 @@ Some of the use cases that we aim to support with UAST are:
 
 ### Current status
 
-Currently, Babelfish is in the process of transition to v2 protocol, new
-node representation and Semantic UAST.
+#### Universal AST coverage
 
-All the beta+ drivers support these new features in the latest version
+Not all the constructs are converted in a language-independent way yet.
+As of 1Q2019, every language driver is expected to only support Identifiers,
+String (literals), Imports, Functions in the [Universal AST schema](uast/uast-specification-v2.md).
+
+That is almost everything that let's one to examine the symbols exported
+by the package, but not the control flow (yet).
+
+Meanwhile, in order to locate constracts that are not covered by UAST
+schema yet - one can look for a `@role` field.
+A [Role](https://godoc.org/gopkg.in/bblfsh/sdk.v2/uast/role#Role) is added
+to every native AST node and it contains a language-independent annotations.
+Underlying tree structure for a constract will be different between languages though.
```

Comment:
```suggestion
However, the tree structure for a role construct may be different between languages.
```

Reply:
I was trying to think of a good, short example we could show to illustrate this, but I couldn't come up with anything that didn't generate half a page of tree.